photon-core/src/main/java/org/photonvision/vision/pipe/impl/DrawCalibrationPipe.java

```diff
@@ -56,6 +56,11 @@ protected Void process(Pair<Mat, List<TrackedTarget>> in) {
         int i = 0;
         for (var target : in.getRight()) {
             for (var c : target.getTargetCorners()) {
+                if (c.x < 0 || c.y < 0) {
+                    // Skip if the corner is less than zero
```

Comment:
The default for undetected points being -1 ya?

Reply:
ya

Reply:
ya